src/node.jl

```diff
@@ -275,30 +296,53 @@ function Bonobo.evaluate_node!(tree::Bonobo.BnBTree, node::FrankWolfeNode)
     time_ref = Dates.now()
     domain_oracle = tree.root.options[:domain_oracle]
 
-    x, primal, dual_gap, active_set = solve_frank_wolfe(
-        tree.root.options[:variant],
-        tree.root.problem.f,
-        tree.root.problem.g,
-        tree.root.problem.tlmo,
-        node.active_set;
-        epsilon=node.fw_dual_gap_limit,
-        max_iteration=tree.root.options[:max_fw_iter],
-        line_search=tree.root.options[:lineSearch],
-        lazy=tree.root.options[:lazy],
-        lazy_tolerance=tree.root.options[:lazy_tolerance],
-        add_dropped_vertices=tree.root.options[:use_shadow_set],
-        use_extra_vertex_storage=tree.root.options[:use_shadow_set],
-        extra_vertex_storage=node.discarded_vertices,
-        callback=tree.root.options[:callback],
-        verbose=tree.root.options[:fwVerbose],
-    )
+  if tree.root.options[:variant] != DICG()
```

Comment:
It would be better without the `if` statement. You can add the parameter `pre_computed_set` to the other variants parameter list, it simply won't be used.